Pipfile

```diff
@@ -33,9 +33,9 @@ opbeat = "*"
 whitenoise = "*"
 bandit = "*"
 coverage = "*"
-astroid = "<1.6"
-pylint = "<1.8"
-prospector = {extras = ["with-vulture"]}
+astroid = "*"
+pylint = "*"
```

Comment:
Wouldn't be better to remove them from our Pipfile, since they're prospector deps (astroid is a pylint dep, to be more precise)?

Reply:
Correct. This way we don't need to deal with these dependencies here.